src/views/Gallery.vue

```diff
@@ -1,13 +1,61 @@
 <template>
-    <div class="container">
-        <h1>This is Gallery page</h1>
+  <section class="gallery-page container">
+    <div class="gallery-page__header">
+      <h2 class="gallery-page__title">Galerija ordinacije Srećkov</h2>
     </div>
+    <div class="row">
+      <div
+        v-for="(src, index) in imgs"
+        :key="index"
+        class="col-md-3 pic"
+        @click="() => showImg(index)"
+      >
+        <img class="gallery__img" :src="src">
+      </div>
+    </div>
+    <vue-easy-lightbox
+      :visible="visible"
+      :imgs="imgs"
+      @hide="handleHide"
+    ></vue-easy-lightbox>
```

Comment:
are you sure that this is correct by linter?

Reply:
njet 💃